### PR TITLE
Add alpha transparency to RadViz #419

### LINKED ISSUE #419: Add alpha transparency to RadViz
To make the RadViz a bit easier to read, we can add optional transparency, set by the user to be able to distinguish regions of more or less density.

### PR DESCRIPTION
**Pull request by lumega**
Fixes #419

Add feature requested in issue #419

[Description]
add optional transparency(alpha), set by the user to be able to distinguish regions of more or less density.

Changes proposed in this pull request:

- add alpha default parameter equals to 1 to keep existing behavior (and pass the tests)
- use alpha in the scatter drawing function.

### CODE REVIEW

**Comment by lwgray** [comment]
Thank you for joining us for the Yellowbrick Sprint at PyCon Cleveland. And also, thank you for submitting your first PullRequest - oh yeah!

We will review this asap. We have been quite busy with all the great contributions from sprinters like yourself.  However, feel free to work on another issue.  We hope you continue to contribute.

File: yellowbrick/features/radviz.py [hunk 1]
```diff
@@ -138,10 +138,11 @@ class RadialVisualizer(DataVisualizer):
     """
```
Comment (lwgray) [inline yellowbrick/features/radviz.py]: Awesome job with this PR.  However, a description of "Alpha" must be included in the doc string.  After this, I will merge it

**Comment by bbengfort** [comment]
@lumega @lwgray I can clean up the docstring so we can get this merged in.

**Comment by lwgray** [comment]
@bbengfort sure go ahead with the addition 👍

**Pull request merged**